### PR TITLE
Updated the readme.md

### PR DESCRIPTION
**Pull request by CyberFlameGO**
**Updated the readme.**
__*Changes:*__
 - Replaced OS X with macOS because of the OS series name change after El Capitan but also added that some versions of OS X are still compatible
 - Added Chromium to " This tool works with Google Chrome and Opera's built-in U2F implementations"

### CODE REVIEW

**Comment by anatolebeuzon** [comment]
Thanks! I think we should go one step further and remove the mention of extensions for Safari and Firefox. macOS Safari 13+ and Firefox 60+ [have built-in support](https://caniuse.com/#search=webauthn) for Webauthn (tested successfully)

**Comment by CyberFlameGO** [comment]
Updated again

**Comment by CyberFlameGO** [comment]
I wonder if we should remove Opera since Chromium is there or reword it to say that some variants of Chromium support it

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ![](https://user-images.githubusercontent.com/1144197/28190263-470a80d2-67e7-11e7-81e6-17895d70bf75.png)
 
-Soft U2F is a software U2F authenticator for OS X. It emulates a hardware U2F HID device and performs cryptographic operations using the OS X Keychain. This tool works with Google Chrome and Opera's built-in U2F implementations as well as with the U2F extensions for [OS X Safari](https://github.com/Safari-FIDO-U2F/Safari-FIDO-U2F) and [Firefox](https://addons.mozilla.org/en-US/firefox/addon/u2f-support-add-on/).
+Soft U2F is a software U2F authenticator for macOS and some versions of OS X. It emulates a hardware U2F HID device and performs cryptographic operations using the macOS Keychain. This tool works with Google Chrome/Chromium, Safari, Firefox & Opera's built-in U2F implementations.
```
Comment (btoews) [inline README.md]: > macOS and some versions of OS X

I'd rather just say `macOS` than bother with mentioning that some older versions may work.

> Google Chrome/Chromium, Safari, Firefox & Opera's

Please spell out `and` instead of using `&` here.

**Review by btoews — COMMENTED** [review]
Thanks for opening this!

File: README.md (hunk 1 above)
Comment (CyberFlameGO) [reply README.md]: gotcha will do

Comment (CyberFlameGO) [reply README.md]: Done.

**Comment by btoews** [comment]
Thanks!
